src/shared-components/accordion/style.ts

```diff
@@ -42,6 +52,8 @@ export const AccordionBox = styled.div<{
 }>`
   ${({ noBorder, isOpen }) => (!noBorder ? getBorderStyle(isOpen) : '')};
 
+  position: relative;
```

Comment:
You can play with this in the dev tools but without it the box-shadow in the focus state is clipped by the border. It's not a total improvement but it's a small thing before I get to the accessibility updates this cycle, which is changing out focus state anyway.